mgradm/cmd/cmd.go

```diff
@@ -42,6 +42,19 @@ func NewUyuniadmCommand() (*cobra.Command, error) {
 		SilenceUsage: true, // Don't show usage help on errors
 	}
 
+	rootCmd.AddGroup(&cobra.Group{
+		ID:    "deploy",
+		Title: "Server Deployment:",
```

Comment:
```suggestion
		Title: L("Server Deployment:"),
```

May be it could be nice to also add a check for this in the `check_localizable` script